app/build.gradle

```diff
@@ -76,6 +76,8 @@ dependencies {
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.lifecycle:lifecycle-extensions:2.1.0'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.1.0'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
```

Comment:
Add dependencies properly we below, you have to put the version number in versions.gradle and then use here. see below as an example.

Reply:
Okay, I will fix these issues and thank you for giving time to review my code.

Reply:
Can you please guide me on how to do this because I have not done it before.